src/test/java/teammates/test/driver/EmailChecker.java

```diff
@@ -40,6 +48,25 @@ public static void verifyEmailContent(String emailContent, String filePathParam)
         }
     }
     
+    /**
+     * Verifies that given {@code emailContent} contains information about {@code inviter}.
+     * 
+     * @param inviter
+     * @param emailContent
+     * 
+     * @see EmailGenerator#generateInstructorCourseJoinEmail()
+     */
+    public static void verifyEmailContainsInviterInfo(AccountAttributes inviter, String emailContent) {
```

Comment:
Remove it altogether. Our email API has a one-to-one mapping between email subject and email content, and where this method is called, only the email subject needs checking. Everything else (contents etc.) is tested at the lower/unit level which you have correctly updated to show the enhancement.